README.md

```diff
@@ -1,56 +1,19 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Portfolio Project
 
-> Description the project.
+> This project content several sections that all of them design via HTML and CSS.
 
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- CSS and HTML
 
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
-
-
-## Getting Started
-
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
-To get a local copy up and running follow these simple example steps.
-
-### Prerequisites
-
-### Setup
-
-### Install
-
```

Comment:
- [ ] Please kindly add this part for your README file, and also for each title give a short description, for example, for `setup` you can explain how to clone your repository locally and how to access the directory :upside_down_face: :upside_down_face: